modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -544,6 +544,19 @@
 /obj/item/clothing/head/avipilot/attack_self(mob/living/carbon/user)
 	adjust_goggles(user)
 
+// Donation reward for NetraKyram
```

Comment:
```suggestion
// Donation reward for NetraKyram - public use allowed via the command vendor
```
just a note so people dont remove it for being a donor item; they have it locked in the loadout to only them, but are fine with it being in the vendor